app/uk/gov/hmrc/accessibilitystatementfrontend/models/AccessibilityStatement.scala

```diff
@@ -30,7 +31,7 @@ case class AccessibilityStatement(
   complianceStatus: ComplianceStatus,
   accessibilityProblems: Option[Seq[String]],
   milestones: Option[Seq[Milestone]],
-  automatedTestingOnly: Option[Boolean],
+  automatedTestingOnly: Boolean,
```

Comment:
Could we have a chat about this change? Normally, I would rightly abhor `Option[Boolean]` rather than just `Boolean`. My only comment is that we did this to keep the YAML parser simple, as we did with other optional values which could be additionally just been considered "empty" (e.g. `[Option[Seq[String]]]` rather than `Seq[String]` and have a default `Seq.empty` rather than `None`.

I'm happy with either pattern BUT I would like it to be consistent.